examples/Channel-Creation/main.tf

```diff
@@ -0,0 +1,10 @@
+provider "octopusdeploy" {
+  address = var.serverURL
+  apikey  = var.apiKey
+  space   = var.space
+}
+
+resource "octopusdeploy_channel" "newChannel" {
+  name            = var.channelName
+  project_id      = "Dev"
```

Comment:
It'd be rare that the user knows the project_id, we should guide them and add a `data` element to look up the project.